website_product_configurator/static/src/scss/tooltip.scss

```diff
@@ -0,0 +1,16 @@
+.tooltip-inner {
```

Comment:
These rules would "leak" out of the product configuration context, potentially reaching other elements using these class names. Even if it's unlikely, it's not a good practice to have global css rules like these.

Instead, we should find where they are used and wrap them with a parent's element class.

For example, if they are only used in elements who are children of a ".product_configurator" div, we could wrap them like so:

```scss
.product_configurator {
    .tooltip-inner {
        // ...
    }
    .tooltip {
        // ...
    }
    // etc..
}
```